### PR TITLE
Fix for Connections UI lockup

### PR DESCRIPTION
**Pull request by humbletim**
Fixes problem where Users having many (>50?) connections experience a UI lockup when accessing the PAL's Connections tab.

This PR disables a console.log statement that appears related to the problem (though it's not entirely clear why logging an overly-large JSON blob actually locks up the UI).

#### Testing
NOTE: for accurate test you'll need to have lots of Connections (for me it started locking up at around 50or so, but that threshold might vary).
* Run this PR build
* Open the PAL
* Click to the Connections Tab
* Verify the results are displayed relatively quickly (one symptom of the lockup is that the busy animation stops spinning).

cc: https://highfidelity.fogbugz.com/f/cases/9953

### CODE REVIEW

File: interface/resources/qml/hifi/Pal.qml [hunk 1]
```diff
@@ -26,6 +26,7 @@ Rectangle {
     // Style
     color: "#E3E3E3";
     // Properties
+    property bool debug: false
```
Comment (zfox23) [inline interface/resources/qml/hifi/Pal.qml]: Please add semicolon.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5535/HighFidelity-Beta-PR11958-34243f423952549d4d703ac4497d5678a0ee8405.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5535/HighFidelity-Beta-PR11958-34243f423952549d4d703ac4497d5678a0ee8405.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5539/HighFidelity-Beta-PR11958-abba85ac5427d4daf429f3a5d9b052e2688fed2a.dmg

**Comment by themelissabrown** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5546/HighFidelity-Beta-PR11958-abba85ac5427d4daf429f3a5d9b052e2688fed2a.dmg

**Comment by zfox23** [comment]
build this please

**Comment by zfox23** [comment]
@thoys tested this, this is a PASS :)

waiting for jenkins

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5551/HighFidelity-Beta-PR11958-abba85ac5427d4daf429f3a5d9b052e2688fed2a.dmg

**Comment by zfox23** [comment]
build this please

**Comment by zfox23** [comment]
gustavo, i have a party to go to! pretty please 👐

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5554/HighFidelity-Beta-PR11958-abba85ac5427d4daf429f3a5d9b052e2688fed2a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5554/HighFidelity-Beta-PR11958-abba85ac5427d4daf429f3a5d9b052e2688fed2a.dmg

**Pull request merged**